packages/storybook8/stories/Components/Notification/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Notification as NotificationComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+
+import { controlsToAdd, hiddenControl } from '../../controlsUtils';
+import { NotificationExample } from './snippets/Notification.snippet';
+
+export { Notification } from './Notification.story';
+
+export const NotificationExampleDocsOnly = {
+  render: NotificationExample
+};
+
+const meta: Meta = {
+  title: 'Components/Notification',
+  component: NotificationComponent,
+  argTypes: {
+    notificationStrings: hiddenControl,
+    notificationIconProps: hiddenControl,
+    autoDismiss: controlsToAdd.isNotificationAutoDismiss,
+    showStackedEffect: controlsToAdd.showNotificationStacked,
+    onClick: hiddenControl,
+    onDismiss: hiddenControl
+  }
```

Comment:
need to add default values and hide the unwanted controls

Reply:
<img width="588" alt="Screenshot 2024-07-25 at 3 28 11 PM" src="https://github.com/user-attachments/assets/82cae982-56c3-427a-8e9f-09bdcc6135f0">